Content.Server/Antag/AntagSelectionSystem.cs

```diff
@@ -214,7 +214,7 @@ public void ChooseAntags(Entity<AntagSelectionComponent> ent, List<ICommonSessio
                     continue;
             }
 
-            MakeAntag(ent, session, def);
+            TryMakeAntag(ent, session, def);
         }
```

Comment:
This is going to run the checking logic twice (its already sorted in GetPlayerPool) and is also going to break a test.

Reply:
reverted